recipe/meta.yaml

```diff
@@ -21,14 +20,14 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.7
+    - python
     # Optional dependencies are numpy, python-lz4, psutil
 
 test:
   requires:
     - pip
-    - python <3.10
-    - pytest
+    - python
```

Comment:
`python` can be removed from `test/requires`

Reply:
Done, thanks!